git-hub

```diff
@@ -759,9 +759,16 @@ class CloneCmd (object):
 	def run(cls, parser, args):
 		upstream = urlparse.urlsplit(args.repository).path
 		if upstream.endswith('.git'): upstream = upstream[:-4]
+		own_repo = True
+		proj = ''
```

Comment:
I don't think this is really necessary, `proj` is only used in the `if '/'` block, where is redefined in the first statement, and in the `if not own_repo` block, which is only used if the previous block was executed. I guess the second block can be inside the `if '/'`.

I don't know, there is something about the logic that is making noise to me but I can't figure exactly what it is :)